modMcf/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
     compile group: 'org.apache.commons', name: 'commons-collections4', version: '4.0'
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.4'
     compile 'org.json:json:20180813'
+    compile 'com.github.ben-manes.caffeine:caffeine:2.6.2'
```

Comment:
can you use 2.8.0?

In 2.7 we made the policy adaptive so that it perform better in LRU-biased workloads.